### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: cookbook/form/form_collections.rst [hunk 1]
```diff
@@ -459,7 +459,7 @@ is added to the ``Task`` class by calling the ``addTag`` method. Before this
 change, they were added internally by the form by calling ``$task->getTags()->add($tag)``.
 That was just fine, but forcing the use of the "adder" method makes handling
 these new ``Tag`` objects easier (especially if you're using Doctrine, which
-we talk about next!).
+we will talk about next!).
```
Comment (WouterJ) [inline cookbook/form/form_collections.rst]: hmm, while you're doing this, can you please also remove the usage of the first person from this sentence? E.g. "especially if you're using Doctrine, which you will learn about next!)"

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Easy win! Thanks Andrew!

**Comment by ifdattic** [comment]
Glad to help!

**Pull request merged**